IFC4x3/Properties/r/RadialSpacing_1TqgA7dOj47v0491hg3bm9/Documentation.md

```diff
@@ -0,0 +1 @@
+The distance between the end of the drillholes.
```

Comment:
I would suggest that if the name of the property is generic that the description should be also generic i. e. "The distance between the ends of cylindrical objects."

Reply:
> I would suggest that if the name of the property is generic that the description should be also generic i. e. "The distance between the ends of cylindrical objects."

This one didn't feel clear when discussing with @michel-rives. Can @hessf-ilf or @jontydunn clarify?